src/webpack.js

```diff
@@ -47,32 +47,39 @@ const ensureDirectoryExists = filePath => {
 
 /**
  * @desc Get meta data by read content file appIcon.png, dappConfig.json
- * @return {void}
+ * @return {object} meta data
  */
 const getDappMetaData = () => {
-  const appIconPath = path.join(process.cwd(), 'appIcon.png');
-  const pathDappConfig = path.join(process.cwd(), 'dappConfig.json');
-  const dappConfig = checkFileExistAndPromptError(pathDappConfig)
-    ? JSON.parse(fs.readFileSync(pathDappConfig, 'utf8'))
+  const pathPackageConfig = path.join(process.cwd(), 'package.json');
+  const packageConfig = checkFileExistAndPromptError(pathPackageConfig)
+    ? JSON.parse(fs.readFileSync(pathPackageConfig, 'utf8'))
     : {};
+  const dappConfig = packageConfig.pangea_dapp ? packageConfig.pangea_dapp : {};
 
-  return dappConfig
-    ? {
-        ...dappConfig,
-        image: checkFileExistAndPromptError(appIconPath)
-          ? base64Encode(appIconPath)
-          : null,
-      }
-    : {};
+  const appIconPath = path.join(process.cwd(), dappConfig.icon_path);
```

Comment:
no need to get the full path from my point of view

Reply:
done